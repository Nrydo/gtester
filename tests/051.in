3887268
937171475
650825496