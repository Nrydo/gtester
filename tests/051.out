4175037704653964